### ISSUE TITLE
Changing order format / adding fields

### ISSUE THREAD

**Issue by chris-belcher** [chris-belcher/joinmarket#45]
This will eventually need to be improved with some ideas we've come up with in the past few weeks. This issue is about collecting together the improvements.

The name of open orders should be changed to 'offers'. An offer is what the maker announces, an order is when a taker and maker(s) agree to do a coinjoin. In other words, an offer is an unfilled order. The concepts are different and need different names.

#29 Add minimum maker count
https://github.com/chris-belcher/joinmarket/issues/9#issuecomment-74422605 something for sendmany
#20 utxo auctioning

**Comment by chris-belcher** [comment]
Perhaps the concepts should actually be called the opposite of what I said; order for announced unfilled coinjoin tx, another name for a filled, partially filled or completed coinjoin.

**Comment by adlai** [comment]
How about using the same names as regular limit order books? "ask" for liquidity advertisements, "bid" for liquidity demands. Since the assets on both sides are the same, bids are just asks with a negative maker fee.

**Comment by chris-belcher** [comment]
I thought in regular matching engines, `ask` and `bid` are both liquidity providing.

**Comment by adlai** [comment]
> I thought in regular matching engines, ask and bid are both liquidity providing.

Correct, but JM is a market with only a single direction of order flow, so one of ask/bid adds liquidity, and the other - removes it.
